README.md

```diff
@@ -83,7 +83,7 @@ binary and is not to be considered a public API.
 * [Primary mkosi git repository on GitHub](https://github.com/systemd/mkosi/)
 * [mkosi — A Tool for Generating OS Images](http://0pointer.net/blog/mkosi-a-tool-for-generating-os-images.html) introductory blog post by Lennart Poettering (2017)
 * [The mkosi OS generation tool](https://lwn.net/Articles/726655/) story on LWN (2017)
-* [mkosi: Building Bespoke Operating System Images](https://media.ccc.de/v/all-systems-go-2023-190-mkosi-building-bespoke-operating-system-images) talk at All Systems Go! 2023
+* [systemd-repart: Building Discoverable Disk Images](https://media.ccc.de/v/all-systems-go-2023-191-systemd-repart-building-discoverable-disk-images) and [mkosi: Building Bespoke Operating System Images](https://media.ccc.de/v/all-systems-go-2023-190-mkosi-building-bespoke-operating-system-images) talks at All Systems Go! 2023
```

Comment:
I'd switch the order and go with the mkosi talk first. The repart talk is a good addition in case people want to change anything around the partitioning.